test/terraform_plan_details_example_test.go

```diff
@@ -0,0 +1,46 @@
+package test
+
+import (
+	"testing"
+
+	"github.com/gruntwork-io/terratest/modules/terraform"
+	"github.com/stretchr/testify/assert"
+)
+
+// An example of how to test information in a terraform plan.  This tests that
+// a resource's name is correctly created from a value in a `for_each`
+// expression.
+func TestTerraformPlanDetailsExample(t *testing.T) {
+	outputPath := "/tmp/terraform-plan.plan"
```

Comment:
NIT: You can use https://golang.org/pkg/io/ioutil/#TempFile to generate a temporary file.